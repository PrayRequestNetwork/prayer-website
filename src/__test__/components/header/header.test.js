import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../../components/header/header'; 

describe('Header Component', () => {
  test('Should Render Header Component', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});