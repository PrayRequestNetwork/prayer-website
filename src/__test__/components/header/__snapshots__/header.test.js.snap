// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component Should Render Header Component 1`] = `
"<header id=\\"home\\" style={{...}}>
  <h1>
    <span className=\\"dropCap\\">
      P
    </span>
    rayer 
    <span className=\\"dropCap\\">
      R
    </span>
    equest 
    <span className=\\"dropCap\\">
      N
    </span>
    etwork
  </h1>
  <section className=\\"content\\">
    <LeaveARequest />
  </section>
</header>"
`;
